app/styles/partials/nav-bar.scss

```diff
@@ -0,0 +1,10 @@
+.item > .ui.blue.button {
```

Comment:
this will apply to any generic item and blue button, Please wrap it so it is specific to the navbar